### PR TITLE
dev into main

### PR DESCRIPTION
**Pull request by hugohabicht01**

### CODE REVIEW

File: nuxt.config.ts [hunk 1]
```diff
@@ -18,21 +19,29 @@ export default defineNuxtConfig({
...
-  ssr: false,
+  ssr: true,
   runtimeConfig: {
     secret: "", // override using NUXT_SECRET in .env
-    github_client_id: "", // same here
-    github_client_secret: "", // same here
     database_url: "", // same here
-    discord_client_secret: "",
-    discord_client_id: "",
+    discord_id: "",
```
Comment (hugohabicht01) [inline nuxt.config.ts:26]: why has this changed?

File: vercel.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "crons": [
+    {
+      "path": "/api/discord",
+      "schedule": "0 18 * * 0"
```
Comment (hugohabicht01) [inline vercel.json:5]: is this working? if so, we need to sort out the discord credentials

File: nuxt.config.ts (hunk 1 above)
Comment (moddedTechnic) [reply nuxt.config.ts:26]: good question, do we have an answer to this?

File: server/api/discord.get.ts
Comment (moddedTechnic) [inline server/api/discord.get.ts:1]: is discord ready to go live?

Comment (moddedTechnic) [reply server/api/discord.get.ts:1]: @MaxAFriedrich

**Review by moddedTechnic — CHANGES_REQUESTED** [review]

File: server/api/discord.get.ts
Comment (hugohabicht01) [reply server/api/discord.get.ts:1]: nope, it isn't

**Review by moddedTechnic — APPROVED** [review]

**Pull request merged**